apps/coscrad-frontend/src/utils/generic-components/presenters/styled-image-in-ui-layout.tsx

```diff
@@ -0,0 +1,6 @@
+import { styled } from '@mui/material';
+
+export const StyledImageInUILayout = styled('img')({
```

Comment:
Not a big fan of the name. I don't think it's possible to have an image that's not in a UI Layout. Is StyledImage missing something? If so, what?

Reply:
I guess when I started on this component it was more specific to an image in, for example, the footer rather than in text content.  But I realize it can just be a `StyledImage`